langchain4j-core/pom.xml

```diff
@@ -9,6 +9,7 @@
         <Xlint>-options</Xlint>
         <!-- TODO: remove enforcer.skipRules -->
         <enforcer.skipRules>requireUpperBoundDeps</enforcer.skipRules>
+        <maven.javadoc.skip>true</maven.javadoc.skip><!-- use dokka for javadocs -->
```

Comment:
Please make sure that javadoc is still deployed to https://docs.langchain4j.dev/apidocs/index.html

Reply:
I see it is still generated under `target/site/apidocs` with `mvn -T4C compile javadoc:aggregate`